src/app/pcasts/controllers/get_create_delete_recommendation_controller.py

```diff
@@ -23,12 +24,20 @@ def content(self, **kwargs):
         [recommendation_schema.dump(r).data for r in recommendations]}
 
     elif request.method == "POST":
-      recommendation = \
-        recommendations_dao.create_recommendation(episode_id, user)
+      blurb = None
+      try:
```

Comment:
blurb = json.loads(body).get('blurb', None)
for the true python one liner experience